packages/react/src/components/TearSheet/tear-sheet.scss

```diff
@@ -0,0 +1,38 @@
+.tear-sheet {
+  height: 100%;
+  background-color: $ui-01;
+
+  & * {
+    max-width: 100%;
+  }
+
+  &--header {
+    padding-left: $spacing-07;
+    padding-right: $spacing-07;
```

Comment:
Can we add a little more padding to the right side when in RTL mode?
![Screen Shot 2021-05-20 at 4 18 12 PM](https://user-images.githubusercontent.com/609466/119043520-1f319a00-b987-11eb-803f-1078036c9f3e.png)

Reply:
Good catch! I've added more padding to the right side of the header. As for the TearSheet content, it will be up to the user implementing the TearSheet. For this particular story, I've added `2rem` of `margin-right` to the **Open 2nd sheet** button.